pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/manager/controller_manager_config.go

```diff
@@ -46,6 +47,12 @@ func (f *ControllerManagerConfig) SetTemplateDefaults() error {
 
 const controllerManagerConfigTemplate = `apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
 kind: ControllerManagerConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: controller-manager-configuration
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
 health:
```

Comment:
created by kustomize 
part-of ->. controller-runtime.sigs.k8s.io
the version is for its api -> v1alpha1